programs/marinade-referral/src/states.rs

```diff
@@ -77,6 +77,12 @@ pub struct ReferralState {
 
     // emergency-pause flag (bool)
     pub pause: bool,
+
+    // fees that will be assigned to referrals, calculated in basis points
+    pub max_operation_fee: u8, // TODO: should this be configurable or a const in program? Does it matter the size of the referral state account?
```

Comment:
the size of the referral state account does not matter. It is good that it is there and it should be configurable by admin only. Also make it 50 bp by default

More over, when creating the referral state account, add 96 bytes to the account size for future expansion

Reply:
+1, this will be done on update of the CLI in PR https://github.com/marinade-finance/liquid-staking-referral-cli/pull/2
It's planned for later update when we finalize the fields arrangement in the struct `ReferralState`